spec/i18n/placeholders_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe "Placeholders in strings" do
+  it "translations preserve placeholders in strings" do
+    dirs = [Rails.root.join('locale')]
```

Comment:
@Fryguy Maybe I don't understand the question here correctly, but I think the ManageIQ
plugins already do provide translations for their stuff. Like here:
https://github.com/ManageIQ/manageiq-providers-amazon/pull/28